### PR TITLE
feat(tests-fuzz): add CreateTableExprGenerator & AlterTableExprGenerator

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

1. add CreateTableExprGenerator
2. add AlterTableExprGenerator

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

## Refer to a related PR or issue link (optional)
#3174

### CODE REVIEW

File: tests-fuzz/src/generator/alter_expr.rs [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        self
+    }
+
+    fn generate_inner(&self) -> Result<AlterTableExpr> {
+        let mut rng = rand::thread_rng();
+        let idx = rng.gen_range(0..3);
+        // 0 -> AddColumn
+        // 1 -> DropColumn(invariant: We can't non-primary key columns, non-ts columns)
+        // 2 -> RenameTable
+        let alter_expr = match idx {
```
Comment (fengjiachun) [inline tests-fuzz/src/generator/alter_expr.rs:54]: Can we add three methods: `add_column`, `drop_column`, `rename_table` ?

File: tests-fuzz/src/generator/create_expr.rs [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+use super::Generator;
+use crate::error::{self, Error, Result};
+use crate::ir::create_expr::{ColumnOption, CreateTableExprBuilder};
+use crate::ir::{
+    self, generate_random_value, Column, CreateTableExpr, PartibleColumn, TsColumn,
+    PARTIBLE_DATA_TYPES,
+};
+
+pub struct CreateTableExprGenerator {
+    columns: usize,
```
Comment (fengjiachun) [inline tests-fuzz/src/generator/create_expr.rs:30]: column_count ?

**Review by fengjiachun — APPROVED** [review]
LGTM

**Comment by WenyXu** [comment]
@fengjiachun @waynexia I refactored the original `AlterExprGenerator` into three generators.

**Review by waynexia — APPROVED** [review]
Believe!

**Pull request merged**